x/auth/legacy/v0_36/migrate.go

```diff
@@ -0,0 +1,14 @@
+// DONTCOVER
+// nolint
+package v0_36
+
+import (
+	v034auth "github.com/cosmos/cosmos-sdk/x/auth/legacy/v0_34"
+)
+
+// Migrate accepts exported genesis state from v0.34 and migrates it to v0.36
+// genesis state. All entries are identical except for validator slashing events
+// which now include the period.
```

Comment:
Where is the period added?

Reply:
that comment doesn't apply here, the only change to auth was removing the FeeCollectorKeeper